lib/cli.rb

```diff
@@ -76,6 +78,31 @@ def run
       end
     end
 
+    command :'ose secret pull' do |c|
+      c.syntax = 'ccli ose secret pull <secret-name>'
+      c.description = 'Selects the current cryptopus folder'
```

Comment:
Add "Only one at a time." As not to confuse people when they want to add multiple secrets as arguments.